### PR TITLE
Runtime: Annotate all fall through switch case 

### PR DESCRIPTION
**Pull request by hhugo**
Based on #1687

### CODE REVIEW

File: runtime/format.js [hunk 1]
```diff
@@ -76,9 +76,12 @@ function caml_parse_format(fmt) {
           i++;
         }
         i--;
+        break;
```
Comment (hhugo) [inline runtime/format.js:79]: @vouillon, I suspect the missing break was a mistake

File: runtime/parsing.js [hunk 2]
```diff
@@ -225,12 +228,14 @@ function caml_parse_engine(tables, env, cmd, arg) {
           cmd = loop;
           continue the_loop;
         }
-      // Fall through
+      // Unreachable
```
Comment (hhugo) [inline runtime/parsing.js:231]: The condition above never "exit normally".

**Comment by hhugo** [comment]
cc @smorimoto

File: biome.json [hunk 3]
```diff
@@ -43,10 +43,9 @@
         "noAssignInExpressions": "off",
         "noControlCharactersInRegex": "off",
         "noDoubleEquals": "off",
-        "noFallthroughSwitchClause": "off",
+
```
Comment (smorimoto) [inline biome.json:46]:
```suggestion
```

**Review by smorimoto — APPROVED** [review]

**Pull request merged**